govcd/api_vcd_test.go

```diff
@@ -49,8 +50,12 @@ type TestConfig struct {
 		SysOrg   string `yaml:"sysOrg"`
 	}
 	VCD struct {
-		Org     string `yaml:"org"`
-		Vdc     string `yaml:"vdc"`
+		Org         string `yaml:"org"`
+		Vdc         string `yaml:"vdc"`
+		ProviderVdc struct {
+			Id               string `yaml:"id"`
```

Comment:
What are these `Id` and `StorageProfileId`?
Foe all the other tests, we refer to entities by name. Could you change it so that we know what a tester should provide to try this new feature?
An example and some comments would be also useful.